### PR TITLE
Change node event hub cardinality

### LINKED ISSUE #429: Switch Node.JS event hubs template over to use batching
This is a more targeted version of: #293 

Specifically @tohling has seen a few customers be more successful with batching for JavaScript functions (see https://github.com/Azure/azure-webjobs-sdk-script/issues/1386#issuecomment-293372407). More investigation is required to understand exactly what factors are at play but in the meantime, users will be more successful with a batch based template.

### PR DESCRIPTION
**Pull request by mamaso**
@soninaren gonna need you on this one, we need to discuss what lights up in the portal / how much of that we want to do :)

resolves #429

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -113,6 +113,26 @@
                     "label": "$eventHubTrigger_connection_label",
                     "help": "$eventHubTrigger_connection_help",
                     "placeholder": "[variables('selectConnection')]"
+                },
+                {
+                    "name": "cardinality",
+                    "value": "enum",
+                    "resource": "EventHub",
```
Comment (soninaren) [inline Bindings/bindings.json]: Remove  "resource": "EventHub". This opening blades for resource selection like storage, eventhub connection

File: Bindings/bindings.json [hunk 2]
```diff
@@ -113,6 +113,26 @@
...
+                            "display": "one"
+                        },
+                        {
+                            "value": "many",
+                            "display": "many"
+                        }
+                    ],
+                    "label": "$eventHubTrigger_cardinality_label",
+                    "help": "$eventHubTrigger_cardinality_help",
+                    "placeholder": "$selectCardinality"
```
Comment (soninaren) [inline Bindings/bindings.json]: Don't need placeholder. It is only for textbox. This would be a dropdown.

File: Bindings/bindings.json [hunk 3]
```diff
@@ -113,6 +113,26 @@
...
+                        {
+                            "value": "one",
+                            "display": "one"
+                        },
+                        {
+                            "value": "many",
+                            "display": "many"
+                        }
+                    ],
+                    "label": "$eventHubTrigger_cardinality_label",
```
Comment (soninaren) [inline Bindings/bindings.json]: Add corresponding values for $eventHubTrigger_cardinality_label and  $eventHubTrigger_cardinality_help in resource files.
Here is a [sample ](https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Resources/Resources.resx#L129).

Comment (mamaso) [inline Bindings/bindings.json]: https://github.com/Azure/azure-webjobs-sdk-templates/pull/430/files?w=1#diff-73189aaa4df7ca5d944ae99dd2d5949aR1149

Didn't show up in the diff, funky.

**Pull request merged**